src/function/table/call/show_sequences.cpp

```diff
@@ -0,0 +1,116 @@
+        int64_t minValue, int64_t maxValue, bool cycle)
+        : name{std::move(name)}, databaseName{std::move(databaseName)}, startValue{startValue},
+          increment{increment}, minValue{minValue}, maxValue{maxValue}, cycle{cycle} {}
+};
+
+struct ShowSequencesBindData : public CallTableFuncBindData {
+    std::vector<SequenceInfo> sequences;
+
+    ShowSequencesBindData(std::vector<SequenceInfo> sequences, std::vector<LogicalType> returnTypes,
+        std::vector<std::string> returnColumnNames, offset_t maxOffset)
+        : CallTableFuncBindData{std::move(returnTypes), std::move(returnColumnNames), maxOffset},
+          sequences{std::move(sequences)} {}
+
+    std::unique_ptr<TableFuncBindData> copy() const override {
+        return std::make_unique<ShowSequencesBindData>(sequences, columnTypes, columnNames,
+            maxOffset);
+    }
+};
+
+static common::offset_t tableFunc(TableFuncInput& input, TableFuncOutput& output) {
+    auto& dataChunk = output.dataChunk;
+    auto sharedState = input.sharedState->ptrCast<CallFuncSharedState>();
+    auto morsel = sharedState->getMorsel();
+    if (!morsel.hasMoreToOutput()) {
+        return 0;
+    }
+    auto sequences = input.bindData->constPtrCast<ShowSequencesBindData>()->sequences;
+    auto numSequencesToOutput = morsel.endOffset - morsel.startOffset;
+    for (auto i = 0u; i < numSequencesToOutput; i++) {
+        auto sequenceInfo = sequences[morsel.startOffset + i];
+        dataChunk.getValueVector(0)->setValue(i, sequenceInfo.name);
+        dataChunk.getValueVector(1)->setValue(i, sequenceInfo.databaseName);
+        dataChunk.getValueVector(2)->setValue(i, sequenceInfo.startValue);
+        dataChunk.getValueVector(3)->setValue(i, sequenceInfo.increment);
+        dataChunk.getValueVector(4)->setValue(i, sequenceInfo.minValue);
+        dataChunk.getValueVector(5)->setValue(i, sequenceInfo.maxValue);
+        dataChunk.getValueVector(6)->setValue(i, sequenceInfo.cycle);
+    }
+    return numSequencesToOutput;
+}
+
+static std::unique_ptr<TableFuncBindData> bindFunc(main::ClientContext* context,
+    TableFuncBindInput*) {
+    std::vector<std::string> columnNames;
+    std::vector<LogicalType> columnTypes;
+    columnNames.emplace_back("name");
+    columnTypes.emplace_back(*LogicalType::STRING());
+    columnNames.emplace_back("database name");
+    columnTypes.emplace_back(*LogicalType::STRING());
+    columnNames.emplace_back("start value");
+    columnTypes.emplace_back(*LogicalType::INT64());
+    columnNames.emplace_back("increment");
+    columnTypes.emplace_back(*LogicalType::INT64());
+    columnNames.emplace_back("min value");
+    columnTypes.emplace_back(*LogicalType::INT64());
+    columnNames.emplace_back("max value");
+    columnTypes.emplace_back(*LogicalType::INT64());
+    columnNames.emplace_back("cycle");
+    columnTypes.emplace_back(*LogicalType::BOOL());
+    std::vector<SequenceInfo> sequenceInfos;
+    auto localDatabaseName = "local(kuzu)";
```

Comment:
Can u refactor `"local(kuzu)"` into a constant and move it into `constants.h`? We are also using it in `ShowTables`.